docs/layouts/shortcodes/generated/rocksdb_configurable_configuration.html

```diff
@@ -62,6 +62,12 @@
             <td><p>Enum</p></td>
             <td>The specified compaction style for DB. Candidate compaction style is LEVEL, FIFO, UNIVERSAL or NONE, and Flink chooses 'LEVEL' as default style.<br /><br />Possible values:<ul><li>"LEVEL"</li><li>"UNIVERSAL"</li><li>"FIFO"</li><li>"NONE"</li></ul></td>
         </tr>
+        <tr>
```

Comment:
The CI failed seems because the doc is not consistent with [RocksDBConfigurableOptions.java](https://github.com/apache/flink/pull/24072/files#diff-162968bbd8c0d2dbfd91b191a97fa012ab1b3c27a329a2581746a66ad8ac76a3), Could you check it ?